agent/xds/testdata/endpoints/connect-proxy-with-chain-and-failover.golden

```diff
@@ -0,0 +1,73 @@
+{
+  "versionInfo": "00000001",
+  "resources": [
+    {
+      "@type": "type.googleapis.com/envoy.api.v2.ClusterLoadAssignment",
+      "clusterName": "db.default.dc1.internal.11111111-2222-3333-4444-555555555555.consul",
+      "endpoints": [
+        {
+          "lbEndpoints": [
+            {
+              "endpoint": {
+                "address": {
+                  "socketAddress": {
+                    "address": "10.10.1.1",
+                    "portValue": 0
```

Comment:
Apparently none of these new or existing golden files have ports set, because `structs.TestNodeService` is used for all of these mock replies from the `v1/health` endpoint and that just is:
```
func TestNodeService(t testing.T) *NodeService {
	return &NodeService{
		Kind:    ServiceKindTypical,
		Service: "web",
	}
}
```

Reply:
That was easy enough to correct.